tarbz2.go

```diff
@@ -20,7 +20,7 @@ func TarBz2(tarbz2Path string, filePaths []string) error {
 	}
 	defer out.Close()
 
-	bz2Writer := bzip2.NewWriter(out)
+	bz2Writer, _ := bzip2.NewWriter(out, nil)
```

Comment:
Why not actually check the error instead of ignoring it? Presumably it may have some non-nil value, perhaps in the future. If it's always going to be nil, `bzip2.NewWriter` wouldn't return it.

I suppose you're trying to make the least number of changes just to update the API and allow this package to compile, but... I can't help but mention this.